config/worker.local.yaml

```diff
@@ -31,6 +31,9 @@ adapters:
   instanceStorage:
     redis:
       url: "redis://localhost:6379/0"
```

Comment:
@caiordjesus you need to add scheduler storage and scheduler cache configs in rooms-api YAML file as well

Reply:
Nice! Just did it!